index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Panda</title>
+    <link rel="stylesheet" href="panda.css">
+</head>
+<body>
+    <div class="nav">
+        <div class="nav_content">
+            <a id="nav1" href="/">
+                <img src="images/nav1.png">
+            </a>
+            <a id="nav2" href="/login">
+                <img src="images/nav2.png">
+            </a>
```

Comment:
❗️ 수정요청
이미지가 아닌 html로 구현하시는 것을 추천드려요~

```suggestion
            <a id="nav2" href="/login">로그인</a>
```